forward_engineering/api.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
 			const _ = dependencies.lodash;
 			let preparedData = {
 				...data,
-				containerData
+				containerData,
+				includeAllModelDefinitionsStatements: true,
```

Comment:
is there a case, where the "preparedData" or similar payload is used without this new flag?

Reply:
Yes, on 
https://github.com/hackolade/Protobuf/blob/ddd30915477d05e6579c45a3bf3cc747de72d16f/forward_engineering/api.js#L81-L85

Reply:
jeez, missed there are different values 🤦🏻